src/odemis/driver/tmcm.py

```diff
@@ -417,7 +417,10 @@ def __init__(self, name, role, port, axes, ustepsize, address=None,
             if sw_rng is not None:
                 if not sw_rng[0] <= 0 <= sw_rng[1]:
                     raise ValueError("Range of axis %d doesn't include 0: %s" % (i, sw_rng))
-                phy_rng = (max(phy_rng[0], sw_rng[0]), min(phy_rng[1], sw_rng[1]))
+                # Make sure the range is a multiple of the ustepsize
```

Comment:
This sounds logical but why is that necessary? Probably due to issues in the field with the tmcm?

Reply:
Good catch. The comment needs to explain *why* (not *what*). Essentially, it's because if it's not rounded and the user moves to the limit, the hardware will round the value, and there is 50% chances it ends up a tiny bit outside the range... which confuses some code (and some users).